Armadillo/src/main/java/com/scribd/armadillo/Reducer.kt

```diff
@@ -261,6 +271,42 @@ internal object Reducer {
                         ))
                         .apply { debugState = newDebug }
             }
+            is OpeningLicenseAction -> {
+                val oldDrm = oldState.drmPlaybackState
+                return oldState
+                    .copy(drmPlaybackState = DrmState.LicenseOpening(action.type ?: oldDrm.drmType))
+            }
+            is LicenseAcquiredAction -> {
+                val oldDrm = oldState.drmPlaybackState
+                return oldState
+                    .copy(drmPlaybackState = DrmState.LicenseAcquired(action.type, oldDrm.expireMillis))
```

Comment:
For my understanding - what's the difference between `LicenseAcquired` and `LicenseUsable`?

Reply:
1. Got one from the remote server.
2. It works and playback should be happening.

Reply:
What needs to happen between 1 and 2?

Reply:
They're different Exoplayer events, - acquisition doesn't necessarily mean that decryption worked.